bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/lib/enums/DeviceType.java

```diff
@@ -24,6 +24,7 @@
     ceiling,
     ceiling1,
     ceiling3,
+    ceiling4,
```

Comment:
If there is no difference between all these ceiling devices, why not just one `DeviceType` ceiling.